templates/packer/ansible/roles/common/tasks/packages.yml

```diff
@@ -0,0 +1,76 @@
+---
+
+- name: Add osbuild-composer repository
+  yum_repository:
+    name: "composer"
+    description: "osbuild-composer commit {{ COMPOSER_COMMIT }}"
+    baseurl: "http://osbuild-composer-repos.s3.amazonaws.com/osbuild-composer/rhel-8-cdn/x86_64/{{ COMPOSER_COMMIT }}"
+    enabled: yes
+    gpgcheck: no
+    priority: "5"
+
+- name: Add osbuild repository
+  yum_repository:
+    name: "osbuild"
+    description: "osbuild commit {{ OSBUILD_COMMIT }}"
+    baseurl: "http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com/osbuild/rhel-8-cdn/x86_64/{{ OSBUILD_COMMIT }}"
+    enabled: yes
+    gpgcheck: no
+    priority: "5"
+
+# We need EPEL for monit
+- name: Add EPEL
```

Comment:
Would be great if we can avoid non-RHEL content... I would say Amazon content would be ok, or anything that we have support for, but we should err on the side of caution.

Could monit and vector be replaced by https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html ?

Reply:
Hm maybe as a standalone project/followup? looks pretty neat tho